app/server/src/main/scala/org/bitcoins/server/DLCRoutes.scala

```diff
@@ -104,5 +104,15 @@ case class DLCRoutes(dlcNode: DLCNodeApi)(implicit system: ActorSystem)
           }
         }
       }
+
+    case ServerCommand("offer-send", arr) =>
+      withValidServerCommand(OfferSend.fromJsArr(arr)) {
+        case OfferSend(remoteAddress, localAddress, message, offerTLV) =>
```

Comment:
I shouldn't have to send the Tor DLC Host address back for this unless there is some use case that makes sense not to use the local one (which is known to the backend already)